pkg/position/range.go

```diff
@@ -48,3 +48,21 @@ func InRange(point ast.Location, theRange ast.LocationRange) bool {
 
 	return true
 }
+
+// RangeGreaterThan returns true if the first range is greater than the second.
```

Comment:
```suggestion
// RangeGreaterOrEqual returns true if the first range is greater than the second.
```
Also, I wonder if we should define/expand what `greater` means. Probably safe to have as is tho.